src/lib/rest/rest.cpp

```diff
@@ -1042,6 +1035,15 @@ static int connectionTreat
   }
 
   // 3. Finally, serve the request (unless an error has occurred)
+  
+  // ServicePath check is delayed to third MHD call 
```

Comment:
Yes I will include new functest and run 'make test', to ensure that all continue working fine